mcp/setup.sh

```diff
@@ -235,6 +240,138 @@ runDeploy() {
   entry
 }
 
+runNewVersion(){
+  echo ""
+  echo "Checking for repository ..."
+  echo ""
+  if [ $(command -v git) ]
+  then
+    cd ..
+    git init
+    git add -A
+    git commit -m "initial commit"
+    echo "New version of: "
+    select module in "AppEngine" "CloudRun"; do
+      case $module in
+        AppEngine )
+          appEngineVersion
+          ;;
+        CloudRun )
+          cloudRunVersion
+          entry
+          ;;
+        * )
+          echo ""
+          echo "Invalid Choice"
+          echo ""
+          entry
+          ;;
+      esac
+    done
+  else
+    echo ""
+    echo "Git not installed, please install"
+    echo ""
+    entry
+  fi
+}
+
+appEngineVersion(){
+  echo ""
+  echo "App Engine Version Setup"
+  echo ""
+  read -r -p "Enter project id: " project
+  read -r -p "Enter the version name/id: " version
+  echo ""
+  echo "Continue with these settings? "
+  echo "    Project ID: $project"
+  echo "    Version: $version "
+  echo ""
+
+  select item in "yes" "no" "cancel"; do
+    case $item in
+      yes )
+        git stash
+        BRANCH="gcp-ae-$version"
+        if [ $(git branch --list "$BRANCH") ]
+        then
+          git checkout "$BRANCH"
+        else
+          git checkout -b "$BRANCH"
+        fi
+        cd terraform
+        terraform workspace new "$BRANCH"
+        terraform init
+        terraform import 'module.mcp.google_app_engine_application.self[0]' "$project" || echo "Import failed /n" && exit
+        versionHelp
+        ;;
+      cancel )
+        echo "Cancelling setup"
+        entry
+        ;;
+      * )
+        appEngineVersion
+        ;;
+    esac
+  done
+}
+
+cloudRunVersion(){
```

Comment:
There should be the ability to pass arguments without interaction

Reply:
I have update to remove a portion of interaction. So running: `mcpadm new-version cloudrun` would take you to the cloudrun function. 
Did you want it to be possible to have no interaction at all? So e.g. being able to pass in `-project-id=<project-id>` or is it fine being prompted for those values?

Reply:
yes